docs/advanced/plugins/index.md

```diff
@@ -22,6 +22,8 @@ redirect_from:
   </p>
 </blockquote>
 
+You can also find official plugins at https://github.com/babel-plugins and other community plugins by searching [npm](https://www.npmjs.com/search?q=babel-plugin-).
```

Comment:
![screen shot 2015-07-01 at 7 07 34 pm](https://cloud.githubusercontent.com/assets/952783/8461442/77604e0e-2024-11e5-8472-f3e261a6dc43.png)

You can click the edit button on the top right of the file in the diff.